apps/web/src/app/globals.css

```
@@ -42,3 +42,47 @@ body {
+  /* Make main content full width */
+  main {
+    margin-left: 0 !important;
+    padding: 0 !important;
+    max-width: 100% !important;
+  }
+  /* Remove dark mode styling */
+  .dark {
+    color-scheme: light;
+  }
+  /* Force light text colors for print, except inline-colored elements and SVGs */
+  *:not([style*="background-color"]):not(svg):not(svg *) {
```

Stylelint (17.4.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)